src/SpookyV2.cpp

```diff
@@ -9,9 +9,11 @@
 //   July 30 2012: I reintroduced the buffer overflow
 //   August 5 2012: SpookyV2: d = should be d += in short hash, and remove extra mix from long hash
 //   Kendon Bell April 30 2019: Added counters to help with skipping first n bytes
+//   Kendon Bell May 04 2019: Deleted original source code that doesn't get used
 
 #include <memory.h>
 #include "SpookyV2.h"
+#include "R.h"
```

Comment:
Why ?

Reply:
Mistake - I had it there to use Rprintf to see when the various bits of the algorithm got hit

Reply:
Just send a follow up commit.  If need be I merge it again.  (I am new to the PR into PR merge business ;-)  )

Reply:
Else I just do it in the final cleanup.